sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsFactory.java

```diff
@@ -1452,7 +1460,18 @@ public boolean apply(@Nonnull String input) {
                   + " but received: " + returnType);
```

Comment:
if needs to be updated to correctly handle Collection<String> and raw Collection type